HISTORY.md

```diff
@@ -11,3 +11,10 @@ Move to new repo + bug fix
 ## [0.3.10] - 2021-07-15
 
 Fix build issue
+
+## [0.3.11] - 2021-07-19
+
+Includes commit - [PR-7](https://github.com/red-hat-storage/rlockertools/pull/7)
```

Comment:
I think that we don't need to include info about each PR in release. Only small info what changed should be enough.

Reply:
I took care of it.